tests/compiler/binary.untouched.wat

```diff
@@ -2121,8 +2115,8 @@
   i64.and
   local.set $5
   local.get $2
-  i64.const 63
-  i64.shr_u
+  i64.const -9223372036854775808
+  i64.and
```

Comment:
Is this worth it, given that the constant is now 8 bytes in the binary instead of 1?

Reply:
Hmm, yeah I also have same concerns about size. I'll revert this change for f64 variant